src/withHydrationSuppress.tsx

```diff
@@ -13,13 +13,18 @@ import { useBundle } from "./useBundle";
 // many other capabilities that we don't require and also a big package dependency, we ported just the
 // needed code.
 
-export const withHydrationSuppress = function <T>(
-  loader: BundleLoader<ComponentType<T>>
-) {
+export const withHydrationSuppress = function <T>({
+  loader,
+  options
+}: {
+  loader: BundleLoader<ComponentType<T>>;
+  options?: BackoffOptions;
```

Comment:
I'd probably rename `options` this to `backoff`, `options` is a bit generic, we could see having other kind of `options` in the future :)